### ISSUE TITLE
IPFS Desktop stuck at "IPFS is Starting" on Windows 10

### ISSUE THREAD

**Issue by sduensin** [ipfs-shipyard/ipfs-desktop#1630]
Please describe what you were doing when this error happened.

Just trying to install and use it.  Stuck on "IPFS is Starting".  Tried rebooting.  Tried reinstalling.  Tried updating.  When I tried to update, it crashed with the following:

**Specifications**

- **OS**: win32
- **IPFS Desktop Version**: 0.12.2
- **Electron Version**: 8.2.5
- **Chrome Version**: 80.0.3987.165

**Error**

```
TypeError: ctx.checkForUpdates is not a function
    at click (C:\Program Files\IPFS Desktop\resources\app.asar\src\tray.js:198:30)
    at MenuItem.click (electron/js2c/browser_init.js:1501:9)
    at Function.executeCommand (electron/js2c/browser_init.js:1762:13)
```

**Comment by jessicaschilling** [comment]
Thank you - we'll include this in our next weekly triage.

**Comment by lidel** [comment]
@rafaelramalho19 sending this one your way :gift:

**Comment by lidel** [comment]
I suspect this could be caused by autoupdate check, which was blocking for some reason.
Should be fixed by #1682 (we made initial update check async)
